content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/10-cockroach-db.md

```diff
@@ -0,0 +1,153 @@
+
+|**Color Code**| **Description**|
+|--------------|--------------|
+|Green| Successfully Deployed|
+|Blue | Under Deployment|
+|Red  | Error State|
+
+
+# Output Variables
+
+The exposed output variables of this service layer may be used in other service layers. These output variables are typically used for connectivity purposes:
+
+| Parameter              | Output Variable                                                                     | Description                                     |
+|------------------------|-------------------------------------------------------------------------------------|-------------------------------------------------|
+| Database Username      | `{{.spectro.app.$appDeploymentName.database-<service-name>.COCKROACHDBMSTR_USERNAME}}` | The database user name.                         |
+| Database User Password | `{{.spectro.app.$appDeploymentName.database-<service-name>.COCKROACHDBMSTR_PASSWORD}}`              | The password of the database user name. |
+| Datbase Name           | `{{.spectro.app.$appDeploymentName.<service-name>.COCKROACHDBMSTR_DB_NAME}}`        | The name of the database.
+| Service Hostname       | `{{.spectro.app.$appDeploymentName.database-<service-name>.COCKROACHDBMSTR_SVC}}`      | The Kubernetes service hostname for the database.                |
+| Service Port           | `{{.spectro.app.$appDeploymentName.database-<service-name>.COCKROACHDBMSTR_SVC_PORT}}` | The exposed ports for the database service.              |
+| Service Namespace       | `{{.spectro.app.$appDeploymentName.database-<service-name>.COCKROACHDBMSTR_SVC_NAMESPACE}}` | The namespace of the service.              |
+
+
+# Database Password
+
+You can get the database secret by reading the content of the Kubernetes secret created for the database user. To retrieve the password for the Redis database, use the following command format. 
+
+<br />
+
+```shell
+kubectl get secret <app-name>-<service-name>-user \
+ --namespace <app-name>-<service-name>-ns --output jsonpath='{.data.password}' | base64 --decode
+```
+
+Replace the values with the respective names.
+
+  * app-name: represents the name of the app provided during the app creation process.
+  * service-name: The name of the service layer in the app profile.
+
+Example: 
+
+- App Name: `app-tion-medon`
+
+- Service Name: `cockroachdb-1`
+
+
+```shell
+kubectl get secret  app-tion-medon-cockroachdb-1-user \
+ --namespace app-tion-medon-cockroachdb-1-ns --output jsonpath='{.data.password}' | base64 --decode
+```
+Output:
+```shell
+.Hr1}%DrA2MFf
+```
+# Next Steps
+
+To learn more about developing with CockroachDB, be sure to check out the [CockroachDB Developer Guide](https://www.cockroachlabs.com/docs/stable/developer-guide-overview.html). The developer guide is a great resource for understanding how to get started with CockroachDB and build applications that are scalable, resilient, and secure.
+
+
+# Resources
```

Comment:
Maybe we should have a style about spacing Resources? This looks good to me. When they're double spaced, it sometimes looks too spaced. It could be we double space if there are 5 or more resources. Single space 5 or less. I could add this to our team meeting agenda.